adr-tools.hcl

```diff
@@ -0,0 +1,17 @@
+description = "A command-line tool for working with a log of Architecture Decision Records (ADRs)."
+
+source = "https://github.com/npryce/adr-tools/archive/refs/tags/${version}.tar.gz"
+strip = 1
+
+binaries = [
+  "src/_adr*",
```

Comment:
Do all of the binaries need to be exposed or would `adr` suffice?

Reply:
If `adr` can't find its own plugins without linking them all in, you might need to add `${root}` to `$PATH`. In the manifest you can try:

```hcl
env = {
  PATH: "${root}:${PATH}",
}
```

(note - I'm not sure where the scripts are so it might need to be a subdirectory under `${root}`)

Reply:
Also FYI you can test this locally in this repo by just `hermit install adr`

Reply:
Good point; looks like just linking `src/adr` is sufficient. Thanks!